kfs-core/src/main/resources/edu/arizona/kfs/gl/gl-resources.properties

```diff
@@ -0,0 +1 @@
+error.gl.GlobalTransactionEdit=Accounting line transaction is not allowed because it violates the following Global Transaction Edit validation rule: Reference Origin Code={0}, Fund Group={1}, Sub-Fund Group={2}, Document Type={3}, Object Type={4}, Object Sub-Type={5}.
```

Comment:
They are keys used to place errors in the error message map by various validations, services, or helpers to indicate where a validation had failed, thus we don't see correlating error message values for them in this file.